src/commands/math/basicSymbols.js

```diff
@@ -140,7 +140,7 @@ var BuiltInOpNames = {}; // http://latex.wikia.com/wiki/List_of_LaTeX_symbols#Na
 var TwoWordOpNames = { limsup: 1, liminf: 1, projlim: 1, injlim: 1 };
 var AutoOpNames = Options.p.autoOperatorNames = { _maxLength: 9 };
 (function() {
-  var mostOps = ('arg deg det dim exp gcd hom inf ker lg lim ln log max min sup'
+  var mostOps = ('arg deg det dim exp gcd hom inf ker lg ln log max min sup'
```

Comment:
I'd prefer not to remove `lim` from the auto-operator names list, since I think the original reason it was there still holds (namely that if you saw `lim a_n` in math, even if `lim` is italicized, you would read it as "limit of `a_n`"). Is the concern here that it won't round-trip? That typing `l`, `i`, and `m` will export as in `\lim`, which when imported becomes `\lim_{ }`? Potentially I could update https://github.com/Learnosity/mathquill/pull/48 so that `\lim` without a subscript afterwards is parsed into the auto-operator name (3 separate letters) instead of into `\lim_{ }`, and only `\lim_x` (subscript in the original LaTeX) has the underset subscript, and we could see if we like that better.